dolittle/Studio
# Store raw data for incoming webhooks

https://app.asana.com/0/1200313386408105/1200309624168622

# How to test?

To run the code cd to Source/BusinessMomentsAdaptorImage and run:
yarn ts-node index.ts
Make sure you have installed the packages; like mongoose.
